stopcovid/dialog/models/events.py

```diff
@@ -239,6 +240,23 @@ def apply_to(self, dialog_state: DialogState) -> None:
         dialog_state.user_profile.opted_out = False
 
 
+class UserCreated(DialogEvent):
+    event_type: DialogEventType = DialogEventType.USER_CREATED
+    name: str
+    unit_id: int
+    employer_id: int
+
+    def apply_to(self, dialog_state: DialogState) -> None:
```

Comment:
Do we need to set `validated` to true? Otherwise, they probably won't be able to interact with the system: https://github.com/opus-training/dialog-engine/blob/master/stopcovid/dialog/engine.py#L196-L202

Reply:
thanks for pointing that out--was wondering about that. also begs the question of whether we should be instantiating the rest of the `CodeValidationPayload` here, too, since we're basically bypassing normal registration for these users. i.e. should we also set `is_demo`, `employer_name`, and `unit_name`

Reply:
Yeah, I'm wondering the same thing. How "full" should these users be? probably pretty full is my guess. Scadmin and the dialog engine already have to have the same schemas for these objects - should we pass those directly, rather than cherry picking fields?

Reply:
i'm leaning towards yes--I don't really see why not? would be curious for @Pastromhaug to weigh in here, too, and if there was a reason why we went with the leaner field set in the PRD

Reply:
I think it would probably just be cleanest for user dialog state to be the same, whether they were created via the normal registration flow or `CreateUser`

Reply:
I wrote that part of the PRD :)

I was outlining the bare minimum of what we need. I think it makes sense to pass objects that adhere to the UserProfile schema

Reply:
The schema of the command could be:

```
{
type: CREATE_USER,
payload: { phone_number: str, user_profile: UserProfile}
}
```

Reply:
okay, gotcha! yeah that sounds great to me. I'll make the updates